admin/app/pagepresser/SimpleHtmlCleaner.scala

```diff
@@ -4,12 +4,12 @@ import org.jsoup.nodes.Document
 
 object SimpleHtmlCleaner extends HtmlCleaner {
 
-  override def canClean(document: Document) = {
+  override def canClean(document: Document): Boolean = {
     document.getElementsByAttribute("data-poll-url").isEmpty &&
     document.getElementById("interactive-content") == null
   }
 
-  override def clean(document: Document, convertToHttps: Boolean) = {
+  override def clean(document: Document, convertToHttps: Boolean): Document = {
```

Comment:
I wonder if this makes any difference if the overridden method already includes a return type.